src/Microsoft.Identity.Client/MsalUiRequiredException.cs

```diff
@@ -40,6 +72,38 @@ public class MsalUiRequiredException : MsalServiceException
         /// <param name="innerException">Represents the root cause of the exception.</param>
         public MsalUiRequiredException(string errorCode, string errorMessage, Exception innerException) : base(errorCode, errorMessage, innerException)
         {
+           
+        }
+
+        /// <summary>
+        /// Classification of the conditional access error, enabling you to do more actions or inform the user depending on your scenario. See https://aka.ms/msal-net-uirequiredexception-classification for details.
+        /// </summary>
+        /// <remarks>This class <see cref="MsalUiRequiredException"/> lists most classification strings as constants. </remarks>
+        public string Classification
```

Comment:
IMO, we should stick to suberror publicliy if we plan to combine this with with other sub errors. If we want to stick to this being related to invalid grant classifications then we should imo name it InvalidGrantClassifications, as we are then clear that we link the protocol to the property. 

As mentioned above, I prefer for now to stick with sub-error as that seems to be more flexible. 

Also, really not sure if this is an MsalUiRequiredException in all cases... as there are suberrors which essentially should be a different type of error as it can't be resulved with us showing Ui)